examples/cms-sanity/now.json

```diff
@@ -0,0 +1,5 @@
+{
+  "env": {
+    "NEXT_EXAMPLE_CMS_DATOCMS_PREVIEW_SECRET": "@next_example_cms_datocms_preview_secret"
```

Comment:
Could you update the secret to `NEXT_EXAMPLE_CMS_SANITY_PREVIEW_SECRET` and `next_example_cms_sanity_preview_secret`?